sys-kernel/ck-sources/metadata.xml

```diff
@@ -2,8 +2,8 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
```

Comment:
Technically speaking, GLEP 76 doesn't care about `<name/>` in metadata.xml, so you can put whatever you want there. The important point is just the e-mail address update **which should have been merged before you changed it on Bugzilla**.